atlassian_jwt_auth/verifier.py

```diff
@@ -16,6 +16,8 @@ def __init__(self, public_key_retriever, **kwargs):
         self._seen_jti = OrderedDict()
         self._subject_should_match_issuer = kwargs.get(
             'subject_should_match_issuer', True)
+        self._check_jti_uniqueness = kwargs.get(
```

Comment:
How about we call this `require_unique_jti`?

Reply:
Sure.